src/test/scala/com/exasol/cloudetl/kafka/ittests/RecordFieldSpecificationIT.scala

```diff
@@ -130,21 +95,38 @@ class RecordFieldSpecificationIT extends KafkaTopicDataImporterAvroIT {
     createCustomTopic(topic)
     val recordTimestamp = 123456L
     withProducer[String, AvroRecord, Unit] { producer =>
-      producer.send(
-        new ProducerRecord[String, AvroRecord](
-          topic,
-          0,
-          recordTimestamp,
-          "record_key",
-          customRecord
-        )
-      )
+      producer.send(new ProducerRecord[String, AvroRecord](topic, 0, recordTimestamp, "record_key", customRecord))
       ()
     }
-    val values = getEmittedValues(
-      "timestamp, value.str_col, key",
-      Seq(classOf[JLong], classOf[Any], classOf[String])
-    )
+    val values = getEmittedValues("timestamp, value.str_col, key", Seq(classOf[JLong], classOf[Any], classOf[String]))
     assert(values === Seq(recordTimestamp, null, "record_key"))
   }
+
+  private[this] def assertJson(actual: String, expected: String): Unit = {
+    assert(JsonMapper.fromJson[JsonNode](actual) === JsonMapper.fromJson[JsonNode](expected))
+    ()
```

Comment:
```suggestion
```
❓

Reply:
Ahh, yes here we are expecting `Unit` as return type, but assert is returning a type. That is very explicitly put return value of unit `()`, otherwise it'll complain that non-unit return value from assert is discarded.